core/src/epicli/data/common/ansible/playbooks/roles/repository/files/repository/Debian/create-offline-repo.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Please remove "repository" directory, there is a new structure, and this file will be placed as repository/files/server/Debian/create-repository.sh , see PR #580